webapp/components/wpt-amend-metadata.js

```diff
@@ -335,22 +346,43 @@ class AmendMetadata extends LoadingState(PathInfo(ProductInfo(PolymerElement)))
     }
 
     for (const key in browserMap) {
-      this.displayedMetadata.push({ product: key, url: '', tests: browserMap[key] });
+      let node = { product: key, url: '', tests: browserMap[key] };
+      // when key (product) is empty, we will set a label field becuase
+      // this is a test-level triage.
+      if (key === '') {
+        node['label'] = '';
+      }
+      this.displayedMetadata.push(node);
     }
   }
 
   handleTriage() {
     const url = new URL('/api/metadata/triage', window.location);
+    const toast = this.shadowRoot.querySelector('#show-pr');
+
+    const triagedMetadataMap = this.getTriagedMetadataMap(this.displayedMetadata);
+    if (Object.keys(triagedMetadataMap).length === 0) {
+      this.selectedMetadata = [];
+      let errMsg = '';
+      if (this.displayedMetadata.length > 0 && this.displayedMetadata[0].product === '') {
```

Comment:
How does this condition relate to the bug URL and label both being empty? It looks like it depends on how `/api/metadata/triage` will fail in this case.

Can the validation instead happen on the client side before submitting? That would also avoid losing the state if one has selected a number of tests and then accidentally clicks submit.

Reply:
> How does this condition relate to the bug URL and label both being empty? It looks like it depends on how `/api/metadata/triage` will fail in this case.
Correct. They have a different error messages.

> Can the validation instead happen on the client side before submitting? That would also avoid losing the state if one has selected a number of tests and then accidentally clicks submit.

Yea I think disabling the `triage` button if empty is one way to go but I have to do a few experiments with the UI